crates/cheatcodes/src/inspector.rs

```diff
@@ -834,6 +842,23 @@ where {
                 }
             }
         }
+
+        // Match the create against expected_creates
+        if let (Some(address), Some(call)) = (outcome.address, call) {
```

Comment:
let's check first if we have any expected create here before we load the account and clone the bytecode